deb/openmediavault/srv/salt/omv/deploy/avahi/files/etc-avahi-avahi-daemon_conf.j2

```diff
@@ -1,19 +1,22 @@
 {%- set use_ipv4 = salt['pillar.get']('default:OMV_AVAHIDAEMON_USE_IPV4', 'yes') -%}
 {%- set use_ipv6 = salt['pillar.get']('default:OMV_AVAHIDAEMON_USE_IPV6', 'yes') -%}
+{%- set allow_ifaces = salt['pillar.get']('default:OMV_AVAHIDAEMON_ALLOW_IFACES', '') -%}
```

Comment:
```suggestion
{%- set allow-interfaces = salt['pillar.get']('default:OMV_AVAHIDAEMON_ALLOW_INTERFACES', '') -%}
```

Reply:
sorry for that shortening - I personally just don't like too long variables.